lib/rules/class-methods-use-this.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview Rule to enforce that all class methods use 'this'.
+ * @author Patrick Williams
```

Comment:
No copyright. We don't need copyrights inside rules, we have one copyright at the root, that covers the whole repository.